js/localization/messages/fr.json

```diff
@@ -289,86 +289,158 @@
-        "dxDiagram-commandShowGrid": "TODO",
-        "dxDiagram-commandSnapToGrid": "TODO",
-        "dxDiagram-commandGridSize": "TODO",
-        "dxDiagram-commandZoomLevel": "TODO",
-        "dxDiagram-commandAutoZoom": "TODO",
-        "dxDiagram-commandSimpleView": "TODO",
-        "dxDiagram-commandCut": "TODO",
-        "dxDiagram-commandCopy": "TODO",
-        "dxDiagram-commandPaste": "TODO",
-        "dxDiagram-commandSelectAll": "TODO",
-        "dxDiagram-commandDelete": "TODO",
-        "dxDiagram-commandBringToFront": "TODO",
-        "dxDiagram-commandSendToBack": "TODO",
-        "dxDiagram-commandLock": "TODO",
-        "dxDiagram-commandUnlock": "TODO",
-        "dxDiagram-commandInsertShapeImage": "TODO",
-        "dxDiagram-commandEditShapeImage": "TODO",
-        "dxDiagram-commandDeleteShapeImage": "TODO",
-        "dxDiagram-unitIn": "TODO",
-        "dxDiagram-unitCm": "TODO",
-        "dxDiagram-unitPx": "TODO",
-        "dxDiagram-dialogButtonOK": "TODO",
-        "dxDiagram-dialogButtonCancel": "TODO",
-        "dxDiagram-dialogInsertShapeImageTitle": "TODO",
-        "dxDiagram-dialogEditShapeImageTitle": "TODO",
-        "dxDiagram-dialogEditShapeImageSelectButton": "TODO",
-        "dxDiagram-dialogEditShapeImageLabelText": "TODO"
+        "dxDiagram-categoryGeneral": "Général",
+        "dxDiagram-categoryFlowchart": "Organigramme",
+        "dxDiagram-categoryOrgChart": "Structure organisationnelle",
+        "dxDiagram-categoryContainers": "Conteneurs",
+        "dxDiagram-categoryCustom": "Personnalisé",
+        "dxDiagram-commandProperties": "Propriétés",
+        "dxDiagram-commandExport": "Exporter",
+        "dxDiagram-commandExportToSvg": "Exporter en SVG",
+        "dxDiagram-commandExportToPng": "Exporter en PNG",
+        "dxDiagram-commandExportToJpg": "Exporter en JPEG",
+        "dxDiagram-commandUndo": "Annuler",
+        "dxDiagram-commandRedo": "Refaire",
+        "dxDiagram-commandFontName": "Nom de la police",
+        "dxDiagram-commandFontSize": "Taille de la police",
+        "dxDiagram-commandBold": "Gras",
+        "dxDiagram-commandItalic": "Italique",
+        "dxDiagram-commandUnderline": "Souligner",
+        "dxDiagram-commandTextColor": "Couleur texte",
+        "dxDiagram-commandLineColor": "Couleur ligne",
+        "dxDiagram-commandFillColor": "Couleur remplissage",
+        "dxDiagram-commandAlignLeft": "Aligner à gauche",
+        "dxDiagram-commandAlignCenter": "Centrer",
+        "dxDiagram-commandAlignRight": "Aligner à droite",
+        "dxDiagram-commandConnectorLineType": "Type de ligne de connexion",
+        "dxDiagram-commandConnectorLineStraight": "Droit",
+        "dxDiagram-commandConnectorLineOrthogonal": "Orthogonal",
+        "dxDiagram-commandConnectorLineStart": "Début de la ligne de connexion",
+        "dxDiagram-commandConnectorLineEnd": "Fin de la ligne de connexion",
+        "dxDiagram-commandConnectorLineNone": "Ligne",
```

Comment:
@belangervince Should it be `Aucun` here?

Reply:
Yes, you're correct. I just commited the change

Reply:
Thank you! Once the unit tests pass, I will merge this PR. The updated French messages will be available with the next DevExtreme release (v19.2.4).

Reply:
Thank you!